src/components/vm/overview/vmOverviewCard.jsx

```diff
@@ -212,7 +212,7 @@ class VmOverviewCard extends React.Component {
                                     <Popover alertSeverityVariant="info"
                                         position="right"
                                         bodyContent={WATCHDOG_INFO_MESSAGE}>
-                                        <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                        <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230704-092041-0b751cdc-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)